index.js

```diff
@@ -1 +1,29 @@
-console.log('hello world');
+import express from 'express';
+import { read } from './jsonFileStorage.js';
+
+const PORT = 3004;
+
+const app = express();
+
+const whenIncomingRequestIndex = (req, res) => {
+  console.log('request came in ');
+
+  read('data.json', (data, error) => {
+    if (error) {
+      console.log('read error', error);
+    }
+
+    const indexOfRecipe  = req.params.index;
+
+    const recipe = data.recipes[indexOfRecipe];
```

Comment:
similar to the above, consider using shorter and concise variable name instead

```suggestion
    const recipeIndex  = req.params.index;

    const recipe = data.recipes[recipeIndex];
```